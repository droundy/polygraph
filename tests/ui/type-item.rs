polygraph::schema!{
    pub type Number = u64;
}

fn main() {
}
