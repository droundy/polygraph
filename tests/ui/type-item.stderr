error: expected `struct` or `enum`
 --> $DIR/type-item.rs:2:9
  |
2 |     pub type Number = u64;
  |         ^^^^
